Orange/data/io_util.py

```diff
@@ -1,5 +1,7 @@
 import subprocess
+import re
```

Comment:
@PrimozGodec Considering we had issues with nltk replacing re in favor of regex, perhaps we should do the same. It could be our users have variables in non-Latin script.

Reply:
Uf, that is true, but I think they have a different case there. I will check if this particular regex works with re on non-Latin. If it works I would go with re since regex still does not provide macOS wheels.

Reply:
@ajdapretnar I checked used regex with texts in more exotic languages from this PR https://github.com/nltk/nltk/issues/2409. It seems regex used here `re.split(r'(\d+)', element)` works well with those characters. In NLTK library have the case with different regex.

Reply:
Perfect! Thanks for checking!